start_admin_sshd.sh

```diff
@@ -97,6 +97,11 @@ if authorized_keys_command_user=$(jq -e -r '.["ssh"]["authorized-keys-command-us
   echo "AuthorizedKeysCommandUser ${authorized_keys_command_user}" >> "${SSHD_CONFIG_FILE}"
 fi
 
+# Populate ciphers with all the ciphers found in user-data
+if ciphers=$(jq -c '.["ssh"]["ciphers"]?' "${USER_DATA}" | tr -d '[]"'); then
```

Comment:
```suggestion
if ciphers=$(jq -r -e -c '.["ssh"]["ciphers"]? | join(",")' "${USER_DATA}" 2>/dev/null); then
```
We should use `-e` so that the `if` can fail if jq does, otherwise we'd print "Ciphers null".  `-r` means we won't have to strip off the quotes.  `join` can replace the `tr`.  (`tr` was changing the return code to 0 anyway, which would have necessitated `set -o pipefail`, but this seems simpler.)